### PR TITLE
feat(statics): enabled bulk withdrawal feature

### PR DESCRIPTION
**Pull request by krupanand-bitgo**
Enabled Bulk withdrawal feature in testnet and production for assets mentioned in below ticket.

Ticket: WIN-3962

### CODE REVIEW

**Review by margueriteblair — APPROVED** [review]

**Review by margueriteblair — CHANGES_REQUESTED** [review]
Tests need to be fixed

**Review by shivpippal — DISMISSED** [review]

**Review by at31416 — DISMISSED** [review]

**Review by OttoAllmendinger — DISMISSED** [review]

**Review by margueriteblair — DISMISSED** [review]

**Review by shivpippal — DISMISSED** [review]

**Comment by krupanand-bitgo** [comment]
PR updated:
Had to resolve merged conflicts after rebase.
Changes to pushed to enable BULK_TRANSACTION feature in testnet and production.

**Review by OttoAllmendinger — DISMISSED** [review]

**Comment by krupanand-bitgo** [comment]
Approval became stale, resolve merge conflicts

**Review by OttoAllmendinger — DISMISSED** [review]

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -158,6 +162,7 @@ const ALGO_FEATURES = [
   CoinFeature.MULTISIG_COLD,
   CoinFeature.BULK_TRANSACTION,
 ];
+const THTETH_TOKEN_FEATURES = [...AccountCoin.DEFAULT_FEATURES, CoinFeature.BULK_TRANSACTION];
```
Comment (ajays97) [inline modules/statics/src/coins.ts:165]: If this is intended for HTETH tokens, then note that HTETH itself is a testnet network, so we can keep it as below
```suggestion
const HTETH_TOKEN_FEATURES = [...AccountCoin.DEFAULT_FEATURES, CoinFeature.BULK_TRANSACTION];
```

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -14538,7 +14686,7 @@ export const coins = CoinMap.fromCoins([
     18,
     '0xee4d03adfca9a902d09f6e3e09dbd5a8a5122fb6',
     UnderlyingAsset['hteth:bgerchv2'],
-    undefined,
+    THTETH_TOKEN_FEATURES,
```
Comment (ajays97) [inline modules/statics/src/coins.ts:14689]:
```suggestion
    HTETH_TOKEN_FEATURES,
```

**Review by ajays97 — COMMENTED** [review]
No test file changes?

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (mohammadalfaiyazbitgo) [reply modules/statics/src/coins.ts:165]: +1

**Review by mohammadalfaiyazbitgo — COMMENTED** [review]
generally lgtm, one comment to address about hteth naming.

**Comment by krupanand-bitgo** [comment]
> No test file changes?

Test Cases added

**Review by zahin-mohammad — COMMENTED** [review]
Changes lgtm, but have you tested these features for all the coins you modified?

**Comment by krupanand-bitgo** [comment]
> Changes lgtm, but have you tested these features for all the coins you modified?

Yes test cases for all these coins.

File: modules/statics/test/unit/coins.ts [hunk 3]
```diff
@@ -657,3 +669,16 @@ describe('Distributed Custody Features', () => {
     });
   });
 });
+
+describe('Bulk Transaction Features', () => {
+  it('Tokens supports Bulk Withdrawal', () => {
```
Comment (ajays97) [inline modules/statics/test/unit/coins.ts:674]: We can condense checks and avoid ifs here in a future PR.

**Review by ajays97 — APPROVED** [review]
Nice work!!

**Review by saravanan7mani — APPROVED** [review]

**Review by lcovar — APPROVED** [review]

**Pull request merged**